Us10

ysrtirak opened the pull request.

ysrtirak committed:
US10 is started

ysrtirak committed:
US10-TC01 is done

ysrtirak committed:
it is done until US10-TC03

sumermehmet2001 reviewed (commented):
you have changed something in currency and Hooks. please agree with Murat. than I will merge it. thx

ysrtirak committed:
US10 changed

ysrtirak committed:
US10 changed

ysrtirak commented:
@sumermehmet2001  

> you have changed something in currency and Hooks. please agree with Murat. than I will merge it. thx

I fixed the conflicts, Please check it again, I talked with Murat and I fixed problems

ysrtirak committed:
done until US10-TC09
